apps/angular/crud/src/app/data-access/loader.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable, computed, signal } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class LoaderService {
+  private apiCount = 0;
+  private isLoadingSubject = signal<boolean>(false);
+  isLoading$ = computed(this.isLoadingSubject);
```

Comment:
$ is mostly for observable. I don't think you should use it for signal 
and computer is not useful here
you can write isLoading = this. isLoadingSubject.asReadonly() is you wnat a readonly signal.